engine/src/graphics/renderer.cpp

```diff
@@ -8,9 +8,8 @@ namespace gen
 {
 
 	Renderer::Renderer(const char * const appName, const u32 appVersion)
-		: m_device(std::make_unique<Device>(appName, appVersion, "Genesis Engine", VK_API_VERSION_1_2)),
-		  m_swapchain(std::make_unique<Swapchain>(Window::getInstance(), *m_device, m_device->getSurface())),
-		  m_commandBufferPool(std::make_unique<CommandBufferPool>(*m_device))
+		: m_device(std::make_unique<Device>(appName, appVersion, "Genesis Engine", VK_API_VERSION_1_3)),
+		  m_swapchain(std::make_unique<Swapchain>(Window::getInstance(), *m_device, m_device->getSurface()))
```

Comment:
Can't the swapchain itself do `device.getSurface()` since it's being passed the device anyway?